java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/InputStream.java

```diff
@@ -60,10 +60,10 @@ public InputStream(
         this(communicator.getInstance(), encoding, new Buffer(buf));
     }
 
-    /** Constructs an InputStream with an empty buffer and the 1.0 encoding. */
-    InputStream(Instance instance) {
+    /** Constructs an InputStream with an empty buffer. */
+    InputStream(Instance instance, EncodingVersion encoding, boolean direct) {
```

Comment:
Unlike the old API, I made direct explicit here. It's possible an internal caller would know it doesn't need a resizable (initially empty) direct buffer.